wyvern/wyvern_typing.py

```diff
@@ -20,4 +20,4 @@
 REQUEST_SCHEMA = TypeVar("REQUEST_SCHEMA", bound=BaseModel)
 RESPONSE_SCHEMA = TypeVar("RESPONSE_SCHEMA", bound=BaseModel)
 
-WyvernFeature = Union[float, str, List[float], None]
+WyvernFeature = Union[bool, int, float, str, List[float], None]
```

Comment:
There might be a breaking change:
currently the logged boolean features are either `1.0` or `0.0`. with this change, it'll become true/false

Reply:
@suchintan need your input.

Reply:
Should we update the logging code to re-cast those to 1.0 or 0.0?

Reply:
one more fix: we should just have float but not int!!!
tested it out:

```
In [1]: from wyvern.entities.feature_entities import FeatureData

In [2]: from wyvern.entities.identifier import Identifier

In [3]: identifier = Identifier(identifier="dfdadf", identifier_type="product")

In [4]: fd = FeatureData(identifier=identifier, features={"a": 5, "dafdas": True, "blabla": 4
   ...: .2})

In [5]: fd
Out[5]: identifier=product::dfdadf features={'a': 5.0, 'dafdas': True, 'blabla': 4.2}

In [6]: fd.json()
Out[6]: '{"identifier": {"identifier": "dfdadf", "identifier_type": "product"}, "features": {"a": 5.0, "dafdas": true, "blabla": 4.2}}'
```